.ci/ansible/Containerfile.j2

```diff
@@ -1,4 +1,4 @@
-FROM {{ ci_base | default("pulp/pulp-ci-centos:" + pulp_container_tag) }}
+FROM {{ ci_base | default("ghcr.io/pulp/pulp-ci-centos:" + pulp_container_tag) }}
```

Comment:
this was my main motivator for this PR, use github container registry instead of docker